pkg/routeagent/controllers/route/route_test.go

```diff
@@ -0,0 +1,35 @@
+package routecontroller
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Route", func() {
+	Describe("Unit test for router.go", func() {
+		var rc RouteController
```

Comment:
Can you name this "routeController"

Reply:
It has to be of type "RouteController" to reflect the struct "RouteController" defined in route.go

Reply:
I meant rename the var 'rc' to routeController for improved readability.

Reply:
Ahh!! I kept rc to be short and its quite easy to remember what it means.

Reply:
To you maybe since you wrote it :) But as we add more and more test cases and more folks reading the code, I prefer added readability over cryptic var names.

Reply:
Done